app/views/certificates/list/_certificate_table.html.erb

```diff
@@ -0,0 +1,30 @@
+<table class="govuk-table">
+  <caption class="govuk-table__caption govuk-heading-m"><%=heading%></caption>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header">ID</th>
+      <th class="govuk-table__header">Usage</th>
+      <th class="govuk-table__header">Enabled</th>
+      <th class="govuk-table__header">Last updated</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <% certificates.each do |certificate| %>
+      <tr class="govuk-table__row">
+          <td class="govuk-table__cell"><%= certificate.id %></td>
+          <td class="govuk-table__cell"><%= certificate.usage %></td>
+          <td class="govuk-table__cell"><%= certificate.enabled %></td>
+          <td class="govuk-table__cell"><%= format_date_time(certificate.updated_at) %></td>
+          <td class="govuk-table__cell">
+              <% if certificate.encryption? && heading.start_with?('Previous') %>
```

Comment:
`heading.start_with?('Previous')` that's a little bit hacky/fragile... considering this is just for our internal debugging we could leave it for now, but would be nice to have something better when we come to building the real UI

Reply:
I agree, we wouldn't be needing this when we look into something better for the UI